runtime/gc_verbose_handler_vlhgc/VerboseHandlerOutputVLHGC.cpp

```diff
@@ -745,6 +735,32 @@ MM_VerboseHandlerOutputVLHGC::getCycleType(UDATA type)
 	return cycleType;
 }
 
+const char *
+MM_VerboseHandlerOutputVLHGC::getConcurrentTerminationReason(J9HookInterface** hook, UDATA eventNum, void* eventData)
```

Comment:
Lets change this to take in stats and remove hook+event data, `MM_ConcurrentPhaseStatsBase stats`

see https://github.com/eclipse/omr/pull/5736#discussion_r558582223